devel/xxdiff/Portfile

```diff
@@ -1,62 +1,100 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
+# -*- coding: utf-8; mode: tcl; c-basic-offset: 4; indent-tabs-mode: nil; tab-width: 4; truncate-lines: t -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           qt5 1.0
 
 name                xxdiff
-version             3.2
-revision            1
+# the current xxdiff version is 5.0b1, which we approach numerically with 4.9.90
+version             4.9.90
```

Comment:
Inventing version numbers that do not exist upstream is not a good idea. If this is a pre-release, it should be in xxdiff-devel.